django/contrib/sessions/backends/base.py

```diff
@@ -87,33 +88,14 @@ def test_cookie_worked(self):
     def delete_test_cookie(self):
         del self[self.TEST_COOKIE_NAME]
 
-    def _hash(self, value):
-        key_salt = "django.contrib.sessions" + self.__class__.__name__
-        return salted_hmac(key_salt, value).hexdigest()
+    def _salt(self):
+        return "django.contrib.sessions" + self.__class__.__name__
```

Comment:
Is the method necessary for this? If it can't be a class attribute then make it a property.